packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import uniqueid from 'lodash.uniqueid';
+import classNames from 'classnames/bind';
+import * as KeyCode from 'keycode-js';
+import styles from './Switch.module.scss';
+
+const cx = classNames.bind(styles);
+
+const SWITCH_STATE = Object.freeze({
+  ON: 'On',
+  OFF: 'Off',
+});
+
+const propTypes = {
+  /**
+    * Whether or not the Switch is enabled ("ON").
+    */
+  isOn: PropTypes.bool,
+
+  /**
+    * If true, the user won't be able to toggle the switch.
+    */
+  isDisabled: PropTypes.bool,
+
+  /**
+    * Text of the label.
+    */
+  labelText: PropTypes.node.isRequired,
+
+  /**
+    * Callback fired when the state is changed. Parameters are (event).
+    */
+  onChange: PropTypes.func,
```

Comment:
Each of the comment block descriptions have improper indentation

Reply:
formatted.